### PR TITLE
Feat/fix dev page

### PR DESCRIPTION
**Pull request by bmikkelsen22**
# Changes introduced

- use generic content page template for development page
- add links
- fix download buttons

## Related issue(s):

- [CSE-32](https://sbacjira.atlassian.net/browse/CSE-32)

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [x] Added new stories for created/modified components (if applicable)
- [x] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [ ] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [x] Assigned yourself to the PR
- [x] Read through the changes in the `Files changed` tab
- [x] Verified that tests were written/modified
- [x] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [x] Checked out the branch and tested changes locally
- [x] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

File: src/components/DevPageContent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+      progress toward college and career readiness. The domain of performance assessment is quite
+      broad, encompassing a range of response tasks. A Smarter Balanced performance task involves
+      significant interaction of students with stimulus materials and/or engagement in a problem
+      solution, ultimately leading to an exhibition of the students’ application of knowledge and
+      skills, often in writing or spoken language. Stimuli include a variety of information forms,
+      as well as an assignment or problem situation.
+    </p>
+    <h4>CAT Item Specifications Overview</h4>
+    <p>
+      Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
```
Comment (puregarlic) [inline src/components/DevPageContent/index.tsx:141]: Is this supposed to be lorem ipsum, or do we have content for this section?

File: src/components/DevPageContent/index.storybook.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  DevAccForItems,
+  DevAddtlItemSpec,
+  DevFullItemSpec,
+  DevContentSpec,
+  DevIntroELAContentSpec,
+  DevIntroMathContentSpec
+} from '.';
+
+storiesOf('Development Page Components', module)
+  .add('Overview', () => <DevOverview />)
```
Comment (puregarlic) [inline src/components/DevPageContent/index.storybook.tsx:17]: Is there a particular reason why these stories are included in their own section, and not under the existing DevelopmentPage section?

**Review by puregarlic — COMMENTED** [review]
Need some clarification.

File: src/components/DevPageContent/index.tsx (hunk 1 above)
Comment (bmikkelsen22) [reply src/components/DevPageContent/index.tsx:141]: It's lorem ipsum in the figma spec... @tnoelcke do we have content for this section yet?

File: src/components/DevPageContent/index.storybook.tsx (hunk 2 above)
Comment (bmikkelsen22) [reply src/components/DevPageContent/index.storybook.tsx:17]: These are sub-components of the dev page. Should they be under the dev page section?

**Review by danvanhorn — APPROVED** [review]
Everything works as expected, this is a much better layout for these static pages.

**Pull request merged**